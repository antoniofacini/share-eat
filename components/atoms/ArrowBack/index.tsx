const ArrowBack = () => (
  <svg aria-label="retornar" height="32" width="32" viewBox="0 0 512 512" fill="none" xmlns="https://www.w3.org/2000/svg">
    <polygon points="352,128.4 319.7,96 160,256 160,256 160,256 319.7,416 352,383.6 224.7,256 " fill="#ffffff" />
  </svg>
)

export default ArrowBack
