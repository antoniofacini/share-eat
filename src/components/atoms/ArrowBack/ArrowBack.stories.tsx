import { Story, Meta } from '@storybook/react/types-6-0'
import ArrowBack from '.'

export default {
  title: 'Atoms/ArrowBack',
  component: ArrowBack,
} as Meta

export const Default: Story = (args) => (
  <ArrowBack {...args} handler={() => console.log('handle')} />
)
