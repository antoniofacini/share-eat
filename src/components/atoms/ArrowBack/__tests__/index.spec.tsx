import { render } from '@testing-library/react'
import ArrowBack from '..'

describe('<ArrowBack />', () => {
  it('should render ArrowBack component', () => {
    const wrapper = render(<ArrowBack handler={() => console.log('handle')} />)
    expect(wrapper).toMatchSnapshot()
  })
})
