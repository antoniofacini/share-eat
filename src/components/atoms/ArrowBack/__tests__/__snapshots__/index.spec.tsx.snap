// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ArrowBack /> should render ArrowBack component 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <svg
          aria-label="retornar"
          fill="none"
          height="32"
          viewBox="0 0 512 512"
          width="32"
          xmlns="https://www.w3.org/2000/svg"
        >
          <polygon
            fill="#ffffff"
            points="352,128.4 319.7,96 160,256 160,256 160,256 319.7,416 352,383.6 224.7,256 "
          />
        </svg>
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <svg
        aria-label="retornar"
        fill="none"
        height="32"
        viewBox="0 0 512 512"
        width="32"
        xmlns="https://www.w3.org/2000/svg"
      >
        <polygon
          fill="#ffffff"
          points="352,128.4 319.7,96 160,256 160,256 160,256 319.7,416 352,383.6 224.7,256 "
        />
      </svg>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
